sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -182,6 +189,8 @@ public virtual async Task<Response<PersonalizerRankResult>> RankAsync(Personaliz
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         public virtual async Task<Response<PersonalizerRankResult>> RankAsync(IEnumerable<PersonalizerRankableAction> actions, IEnumerable<object> contextFeatures, CancellationToken cancellationToken = default)
         {
+            Models.PersonalizerClientProperties result = await ClientConfigurationRestClient.PostAsync(cancellationToken).ConfigureAwait(false);
+            Console.WriteLine("InternalId: " + result.ApplicationID + "\nStorageBlobUri: " + result.ModelBlobUri + "\nLearningMode: " + result.ProtocolVersion + "\nExploration Percentage: " + result.InitialExplorationEpsilon + "\ninitialCommandline: " + result.InitialCommandLine);
```

Comment:
Do not write to console in production code. If something needs to be output, it should be done through a logger. For this particular case, I would suggest removing entirely.

Reply:
TIP: When you want to print the entire object is usually best to use JSON serialization. `Console.Write(JsonConvert.SerializeObject(result));`